src/i965_drv_video.c

```diff
@@ -1431,8 +1431,23 @@ i965_CreateConfig(VADriverContextP ctx,
         if (attrib_found) {
             uint32_t enc_packed_attribs = i965_get_enc_packed_attributes(ctx, profile, entrypoint);
 
-            if (!(attrib_found->value & enc_packed_attribs))
-                vaStatus = VA_STATUS_ERROR_INVALID_VALUE;
+            if (enc_packed_attribs == VA_ATTRIB_NOT_SUPPORTED) {
+                if (attrib_found->value != VA_ATTRIB_NOT_SUPPORTED) {
+                    i965_log_info(ctx, "vaCreateConfig: invalid EncPackedHeaders attribute %#x: "
+                                  "packed headers are not supported.\n", attrib_found->value);
+                    vaStatus = VA_STATUS_ERROR_INVALID_VALUE;
+                }
```

Comment:
If the driver doesn't support packed header at all, application should not pass VAConfigAttribEncPackedHeaders to the driver. so I think the driver should return error no matter what is attrib_found->value